devops/deploy/spin-up-script.sh

```diff
@@ -1,25 +1,55 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+# Just a basic script to spin up listening servers on different ports in dynamic port range
+# Setting up variables
 PSTART=${1:-50000}
 PEND=${2:-50050}
-# Just a basic script to spin up listening servers on different ports in dynamic port range
+PEER_FILE="peer_list1.txt"
 IP=127.0.0.1
 COUNT=0
 ((NUM_PORTS=${PEND}-${PSTART}))
 
+# Making keys directory to hold public/private keys if not exist
+if [ -d keys/ ]
+then
+	echo "Directory keys exists."
+else
+	mkdir keys/
+fi
+
+# Making Peer file if it does not exist
+if [ -f $PEER_FILE ]
+then
+	echo "Peer List File exists"
+else
+	touch $PEER_FILE
+	chmod 600 $PEER_FILE
+fi
+
 echo "Total number of ports: $NUM_PORTS"
 export PYTHONPATH=../../src/main/py
 
 port=$PSTART
 
-while [ ${COUNT} -lt ${NUM_PORTS} ]
+# Loop for spinning up
+while [ ${COUNT} -le ${NUM_PORTS} ]
 do
   #Check for Open ports, if open then OK, else continue
   if ! netstat -apn | grep -q "$IP:$port" 
   then
+    # Generating keys
+    ssh-keygen -t ed25519 -f "keys/node${COUNT}" -N ""
+
+    # Cutting the keys from the key files
+    private_key=$(cat keys/node${COUNT} | awk -F '-----' '{print $1}' | tr -d '\n')
+    public_key=$(cat keys/node${COUNT}.pub | awk -F '-----' '{print $1}' | tr -d '\n')
+    
```

Comment:
Yes... I don't know but apparently it is not registering the separating newlines as '\n' so I can't do the awk with newlines. So I've thought about separating using -----. Could be because of my wrong options though, which I may try again. O.O